docs/images/k8gb-collaboration.svg

```diff
@@ -0,0 +1,4 @@
y="752" fill="#FFFFFF" font-family="Helvetica" font-size="22px" text-anchor="middle" font-weight="bold">6</text></switch></g><rect x="1339" y="724" width="449" height="42.5" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe flex-start; width: 447px; height: 1px; padding-top: 727px; margin-left: 1341px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: left;"><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">The records are shared between distributed k8gb's cross all regions. All checks repeats cyclically and all changes are gradually propagated to individual clusters.   </div></div></div></foreignObject><text x="1341" y="741" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="14px">The records are shared between distributed k8gb's cross all regi...</text></switch></g><rect x="1293.53" y="95.5" width="499.47" height="80" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 497px; height: 1px; padding-top: 136px; margin-left: 1296px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: left;"><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;"><i>* The following diagrams describe the collaboration between k8gb regions (clusters). The diagram is only for easy understanding of the collaboration between clusters, not step by step what exactly is happening.  The runtime sets the steps in random order due to TTL, and the complete decentralization of all components.    </i></div></div></div></foreignObject><text x="1296" y="139" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px">* The following diagrams describe the collaboration between k8gb regions (clusters)...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
The diagram description under the title still reads "Internal k8gb components and relation between them".
Should it still be the case?
![image](https://user-images.githubusercontent.com/502695/193654513-821fabfb-e889-498c-9a96-71dae2bae963.png)